pkg/master/controller.go

```diff
@@ -131,11 +131,13 @@ func (c *completedConfig) NewBootstrapController(legacyRESTStorage corerest.Lega
 	}
 }
 
+// PostStartHook only runs Start currently
```

Comment:
PostStartHook initiates the core controller loops that must exist for bootstrapping.

Reply:
thx for your replenishment. done.